test/spec/synchronousVersioningSpec.js

```diff
@@ -225,16 +229,16 @@ if (typeof Benchmark !== 'undefined') {
 
   },{
     onStart: function(){
-      banchQueue = new JSONPatchQueueSynchronous("/version",function(){});
       obj = {foo: 1, baz: [
         {qux: 'hello'}
       ]};
+      banchQueue = new JSONPatchQueueSynchronous(obj, "/version",function(){});      
       remoteCounter = 1;
       localCounter = 0;
     }
   });
   suite.add(suite.name + ' queue received operation sequence', function () {
-    banchQueue.receive(obj, [
+    banchqueue.receive([
```

Comment:
what's `batchqueue`? shouldn't it be `banchQueue`?

Reply:
O.O No idea how this happened!